server/cla.go

```diff
@@ -10,16 +10,15 @@ import (
 	"net/http"
 	"strconv"
 	"strings"
-	"time"
 
 	"github.com/google/go-github/v32/github"
 	"github.com/mattermost/mattermost-mattermod/model"
 	"github.com/mattermost/mattermost-server/v5/mlog"
 )
 
-func (s *Server) handleCheckCLA(ctx context.Context, pr *model.PullRequest) error {
+func (s *Server) handleCheckCLA(ctx context.Context, pr *model.PullRequest) (bool, error) {
```

Comment:
Might be good to have a godoc here explaining why the boolean return is needed.

Reply:
Good suggestion